### ISSUE TITLE
iFrame force light/dark mode (prefers-color-scheme)

### ISSUE THREAD

**Issue by jonathanmmm** [nextcloud/calendar#3416]
Hi,

I try to implement nextclouds calendar via iframe into a website, the website is completely held in white, but as my windows settings are set to dark, the iframed calendar is black, which doesn't match the theme.

I tried with like changing all css for iframes, but it takes much time and discolors other.

I propose an enhancement when making iframe like ?force_color_scheme=light

Think it has to to with this pull
https://github.com/nextcloud/server/pull/21366

don't know, maybe it is more of an server issue

**Comment by tcitworld** [comment]
The theme follows `prefers-color-scheme` as long as you are logged-out, otherwise it's your personal accessibility settings that applies. Try in a private navigation window.

Anyway, this is handled in server as you said, not much we can do here.

**Comment by jeekajoo** [comment]
Hello @jonathanmmm @tcitworld

I second this feature request

I think it's a problem when you want to embed the Nextcloud calendar into a website that cannot follow `prefers-color-scheme`.

![screenshot_20211113-233807](https://user-images.githubusercontent.com/1882534/141661043-99c24f42-cf5e-44ad-83ea-728213a9248d.png)

Thanks for your attention

**Comment by jonathanmmm** [comment]
> Hello @jonathanmmm @tcitworld
> 
> I second this feature request
> 
> I think it's a problem when you want to embed the Nextcloud calendar into a website that cannot follow `prefers-color-scheme`.
> 
> ![screenshot_20211113-233807](https://user-images.githubusercontent.com/1882534/141661043-99c24f42-cf5e-44ad-83ea-728213a9248d.png)
> 
> Thanks for your attention

That's exactly my scenario, I tried to embedded it into wordpress and this website should allways be light/white and not dark and the dark calendar in the middle just looks off. I tried changing the whole dark css to light css, but using F12 in a browser to edit it, is not the best idea, as forms, etc. are also changed. So if I have to replace the whole dark with light theme.

Until dark/light theme get new stuff and it breaks, so something like this mode, would be much better.
But I will link it into the server, as it is not a calendar only thing.
